test/integration/tests/print.sh

```diff
@@ -93,4 +93,17 @@ if [ $? -eq 0 ]; then
   exit 1
 fi
 
+# TSS Privkey
+tpm2 createprimary -G rsa -C o \
+-a "fixedtpm|fixedparent|sensitivedataorigin|userwithauth|restricted|decrypt|noda" \
+-c prim.ctx
+
+tpm2 create -C prim.ctx -u mykey.pub -r mykey.priv
+
+tpm2 encodeobject -C prim.ctx -u mykey.pub -r mykey.priv -o mykey.pem
+
+tpm2 print -t TSSPRIVKEY_OBJ mykey.pem
+
+tpm2 print -t TSSPRIVKEY_OBJ -f pem mykey.pem > abc.pem
```

Comment:
abc.pem needs to be checked if it is a valid pem.
Use tsskeypriv.pem instead of mykey
Use public_key.pem instead of abc.pem